package.json

```diff
@@ -41,14 +43,17 @@
   "devDependencies": {
     "@types/node": "^13.1.0",
     "assert-diff": "^3.0.1",
+    "dtslint": "^3.6.14",
     "eslint": "^6.8.0",
     "mocha": "^7.0.0",
     "muk-prop": "^2.0.0",
     "muk-require": "^1.2.0",
     "nock": "^12.0.0",
     "nyc": "^15.0.0",
     "sinon": "^9.0.0",
-    "stream-equal": "~1.1.0"
+    "stream-equal": "~1.1.0",
+    "tslint": "^6.1.2",
+    "typescript": "^3.9.7"
```

Comment:
isn't dtslint the only needed dependency here, since it installs typescript and tslint according to its page https://www.npmjs.com/package/dtslint

Reply:
kinda yes but kinda no
* it does install tslint as a dependencie
* typescript is a peer- & devDependencies
  those do NOT get installed automatically - that feature got remove ~2 years ago